lib/livebook.ex

```diff
@@ -174,6 +174,20 @@ defmodule Livebook do
     end
   end
 
+  @doc """
+  Gracefully stops Livebook.
+  """
+  def shutdown do
```

Comment:
`Livebook` is the only public module and it only includes functions that we find are necessary for other projects, specifically `nerves_livebook`. So let's move this elsewhere for now, could be `Livebook.Config`.